internal/librarian/flags.go

```diff
@@ -51,12 +51,8 @@ func addFlagEnvFile(fs *flag.FlagSet, cfg *config.Config) {
 	fs.StringVar(&cfg.EnvFile, "env-file", "", "full path to the file where the environment variables are stored. Defaults to env-vars.txt within the output")
 }
 
-func addFlagGitUserEmail(fs *flag.FlagSet, cfg *config.Config) {
-	fs.StringVar(&cfg.GitUserEmail, "git-user-email", "noreply-cloudsdk@google.com", "Email address to use in Git commits")
-}
-
-func addFlagGitUserName(fs *flag.FlagSet, cfg *config.Config) {
-	fs.StringVar(&cfg.GitUserName, "git-user-name", "Google Cloud SDK", "Display name to use in Git commits")
+func addFlagPushConfig(fs *flag.FlagSet, cfg *config.Config) {
+	fs.StringVar(&cfg.PushConfig, "push-config", "Google Cloud SDK:noreply-cloudsdk@google.com", "Display name and email address to use in Git commits")
```

Comment:
> "Display name and email address to use in Git commits"

Can we also spell out the format we expect for name and email address in the usage description?  
This code is expecting "display name:email", which is slightly different from the [go/librarian:cli-reimagined](http://goto.google.com/librarian:cli-reimagined) example. I personally think it is fine as long as we have it clearly documented